browseable/gen-index-page.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
```

Comment:
missing license header? I think such code should be under an source code license and not the "Community Specification License 1.0"